docs/examples/plot_lcpl_explainer.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""
+=========================================
+Explaining Local Classifier Per Level
+=========================================
+
+A minimalist example showing how to use HiClass Explainer to obtain SHAP values of LCPL model.
+A detailed summary of the Explainer class has been given at Algorithms Overview Section for :ref:`Hierarchical Explainability`.
+"""
+from sklearn.ensemble import RandomForestClassifier
+import numpy as np
+from hiclass import LocalClassifierPerLevel, Explainer
+
+# Define data
+x_train = np.array(
+    [
+        [40.7, 1.0, 1.0, 2.0, 5.0, 2.0, 1.0, 5.0, 34.3],
+        [39.2, 0.0, 2.0, 4.0, 1.0, 3.0, 1.0, 2.0, 34.1],
+        [40.6, 0.0, 3.0, 1.0, 4.0, 5.0, 0.0, 6.0, 27.7],
+        [36.5, 0.0, 3.0, 1.0, 2.0, 2.0, 0.0, 2.0, 39.9],
+    ]
+)
+x_test = np.array([[35.5, 0.0, 1.0, 1.0, 3.0, 3.0, 0.0, 2.0, 37.5]])
+y_train = np.array(
+    [
+        ["Gastrointestinal", "Norovirus", ""],
+        ["Respiratory", "Covid", ""],
+        ["Allergy", "External", "Bee Allergy"],
+        ["Respiratory", "Cold", ""],
+    ]
+)
```

Comment:
can you please use the platypus dataset? I believe these lines are not adding any useful information for the user.